src/js/components/repositories/listItem.tsx

```diff
@@ -1,29 +1,49 @@
-import Card from '@salesforce/design-system-react/components/card';
+import VisualPicker from '@salesforce/design-system-react/components/visual-picker';
+import VisualPickerLink from '@salesforce/design-system-react/components/visual-picker/link';
+import i18n from 'i18next';
 import React from 'react';
-import { Link } from 'react-router-dom';
 
 import { Repository } from '@/store/repositories/reducer';
 import routes from '@/utils/routes';
 
 const RepositoryListItem = ({ repository }: { repository: Repository }) => (
-  <Link
-    to={routes.repository_detail(repository.slug)}
+  <VisualPicker
+    size="large"
+    id={repository.slug}
     className="slds-text-link_reset
       slds-p-around_small
       slds-size_1-of-1
       slds-medium-size_1-of-2
       slds-large-size_1-of-3"
   >
-    <Card heading={repository.name} bodyClassName="slds-card__body_inner">
-      {repository.description_rendered ? (
-        <div
-          className="truncate-children slds-p-top_x-small"
-          // This description is pre-cleaned by the API
-          dangerouslySetInnerHTML={{ __html: repository.description_rendered }}
+    <VisualPickerLink
+      href={routes.repository_detail(repository.slug)}
+      icon={
+        <img
+          src={
+            repository.repo_image_url
+              ? repository.repo_image_url
+              : 'https://via.placeholder.com/75'
+          }
+          alt={`${i18n.t('image for')} ${repository.name}}`}
         />
-      ) : null}
-    </Card>
-  </Link>
+      }
+      title={repository.name}
+      className="bar"
+      links
+      description={
+        repository.description_rendered ? (
+          <div
+            className="truncate-children slds-p-top_x-small"
+            // This description is pre-cleaned by the API
+            dangerouslySetInnerHTML={{
+              __html: repository.description_rendered,
+            }}
+          />
+        ) : null
+      }
```

Comment:
@jgerigmeyer this component is expecting a string for this `description` prop and throws a `proptype` warning since we are setting the html for markdown.  Curious if this can be suppressed/ignored since it doesn't cause a runtime error, or if you'd have an idea for a workaround.

```react_devtools_backend.js:6 Warning: Failed prop type: Invalid prop description of type object supplied to SLDSVisualPickerLink, expected string.```

Reply:
Yeah, good call. That issue, combined with the fact that we're really not using these like vertical visual pickers at all, led me to just replace the `VisualPicker` components with custom markup following the SLDS guidelines.